srcpkgs/ardor/files/bin/ardor

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cd /usr/lib/ardor && bash run.sh
```

Comment:
imo you can just the content of these binaries as the runit runfiles

Reply:
Yeah, the runit files are not much shorter. I just did it the same way as I did with nxt (which is basically the same (older version 1.x))